cmd/tether/attach.go

```diff
@@ -467,21 +487,32 @@ func (t *attachServerSSH) globalMux(reqchan <-chan *ssh.Request) {
 	// ContainersReq will close this channel
 	t.askedAndAnswered = make(chan struct{})
 
+	var pendingFn func()
 	for req := range reqchan {
-		var pendingFn func()
-		var payload []byte
+		payload := []byte{}
 		ok := true
 
 		log.Infof("received global request type %v", req.Type)
 
 		switch req.Type {
 		case msgs.ContainersReq:
-			keys := make([]string, len(t.config.Sessions))
+			keys := make([]string, len(t.config.Sessions)+len(t.config.Execs))
```

Comment:
are you sure you want to allocate all strings? you will end up with an array that has many empty elements at the end if there are not active elements exist.